src/ext/ddtrace.c

```diff
@@ -226,6 +233,26 @@ static PHP_FUNCTION(dd_trace_forward_call) {
 #endif
 }
 
+static PHP_FUNCTION(dd_trace_cfg) {
```

Comment:
What are your thoughts on renaming this to:

```suggestion
static PHP_FUNCTION(dd_trace_env_config) {
```

Reply:
Its whole 7 characters more to type :D But `dd_trace_env_config` looks good to me 👍